Makefile

```diff
@@ -116,7 +116,7 @@ custom-rpms:
 	@echo "Installing rpms"
 	RPMS_SRC="$(RPMS_SRC)" hack/custom_rpms.sh
 
-test-bin:
+test-bin: init-git-submodules
```

Comment:
does this (actually) run correctly only once if submodules are not inited, right?

Reply:
This will not fail when they are already initiated.
Anyway this "test-bin" target will only be used by the cnf-tests image builder now.

Reply:
Are you ok with overriding gcp?

Reply:
It really runs every time either the submodules are already initiated or not, but running it multiple times does not change the output, so it should be safe.